lisa/util/__init__.py

```diff
@@ -657,3 +659,18 @@ def strip_strs(obj: Any, fields: List[str]) -> Any:
             value = value.strip() if isinstance(value, str) else value
             setattr(obj, field, value)
     return obj
+
+
+def check_till_timeout(
+    func: Callable[..., Any],
+    timeout_message: str,
+    timeout: int = 20,
```

Comment:
Give 60 as the default value, so at least one call doesn't need to specify it.

Reply:
updated